shell/run-dev.sh

```diff
@@ -69,6 +71,6 @@ __EOF__
 
 # Work-around for problem where Meteor's bundled npm prefers the system gyp
 # over its own bundled version, and the system gyp doesn't work.
-export PYTHONPATH=$HOME/.meteor/tools/latest/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib
+export PYTHONPATH=$("$SCRIPT_DIR/../find-meteor-dev-bundle.sh")/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib
 
 exec meteor run -p $PORT --settings $SETTINGS
```

Comment:
Meteor itself needs to run inside the `shell` directory. Maybe just `cd` there from the beginning, rather than for each command?